lib/engine.js

```diff
@@ -460,9 +460,9 @@ module.exports = function container (get, set, clear) {
               if (s.product.max_size && Number(size) > Number(s.product.max_size)) {
                 size = s.product.max_size
               }
-              if (s.buy_order && so.max_slippage_pct) {
+              if (s.buy_order && so.max_slippage_pct != null) {
```

Comment:
shouldn't this be `!== null`?

Reply:
Short version of `!== null || !== undefined`